tfjs-backend-webgl/src/gpgpu_math.ts

```diff
@@ -253,6 +254,7 @@ export function runProgram<T extends Tensor, K extends Tensor>(
         outTex.texture, outTexShape[0], outTexShape[1]);
   }
   gpgpu.setProgram(binary.webGLProgram);
```

Comment:
should this two line be the same as just call gpgpu.buildVao(binary.webGLProgram)?

Reply:
IICU, `buildVao` is to initialize VAO, while here it is using initialized VAO. Specifically, VAO initialization has to bind all attributes, while using initialized VAO does not need to.